infer/src/backend/InferAnalyzeJson.mli

```diff
@@ -0,0 +1,13 @@
+(*
+ * Copyright (c) 2009-2013, Monoidics ltd.
```

Comment:
Please delete this line, it's only present on older source code where appropriate but new code should be just (c) Facebook.

Reply:
> Please delete this line, it's only present on older source code where appropriate but new code should be just (c) Facebook.

Deleted